webapp/src/pages/welcome/welcomePage.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import './welcomePage.scss'
+
+const WelcomePage = React.memo(() => (
+    <div className='WelcomePage'>
+        <h1> TEST 123 123 123 </h1>
```

Comment:
Let's remove the `<h1>` element and test string - an empty div is ok.

Ideally, @asaadmahmood, could you make your changes in this PR, then merge it all at once? Or we should at least add a link to `/dashboard`, so users don't get stuck here, even if it's temporary.